packages/webapp/src/inductions/components/induction-journeys/inviter-witnesses/invite-form.tsx

```diff
@@ -1,5 +1,4 @@
 import { FormEvent, useState } from "react";
-
```

Comment:
so, my rationale in imports is split the third party packages from our own packages... I always try to split in three, or even four, blocks:
```tsx
// import from npm packages 1
// import from npm packages 2
// import from npm packages 3

// import from our global packages, eg: "_app" 1
// import from our global packages, eg: "_app" 2
// import from our global packages, eg: "_app" 3

// import from siblings & relational files: eg "./ or "../xpto" 1
// import from siblings & relational files: eg "./ or "../xpto" 2
// import from siblings & relational files: eg "./ or "../xpto" 3

// import our own static assets, eg: svg, css, img etc. 1
// import our own static assets, eg: svg, css, img etc. 2
// import our own static assets, eg: svg, css, img etc. 3
```

If you don't like it, and prefer to keep them mixed, it's fine for me. I usually find this separation useful for reading, easy to quick scan and understand dependencies.

Reply:
I like your rationale. I'll comb through the files in this PR and adjust to follow it. 👍